### DISCUSSION TITLE
Shouldn't the Windows .zip disable updates by default?

### DISCUSSION THREAD

**Discussion by GitMensch** [VSCodium/vscodium#2173] [Polls]
So far there's only an installation for Windows via .msi which has updates disable and I think the .zip should do it that way as well (no secondary .zip needed in my opinion).

... after all, it is a _manual_ installation.

Also: I'm not sure if the auto-update does work at all in that environment, if if it _does_ work "somehow" with the .zip distribution, then there could be a reason to provide both versions.

**Comment by daiyam** [comment]
For that case, the update is providing a `.zip` as download, isn't it?

**Comment by ghost** [comment]
I voted keep it the same.

You can just build the zip to disable updates by default. Thanks to this VSCodium repo.

Also good job expanding over to RISC-V land even if it's too early for something like Electron to be used on one single dev computer.

Any recommendations for a VSCdoium fork that has stuff like 
- [ ] https://github.com/zokugun/MrCode?tab=readme-ov-file#patched-settings
- [ ] https://github.com/zokugun/MrCode?tab=readme-ov-file#patched-commands
- [ ] https://github.com/zokugun/MrCode?tab=readme-ov-file#patched-features
- [ ] Uses  open-vsx.org by default if configured by the user, not by default
- [ ] https://open-vsx.org/extension/zokugun/vsix-manager

I won't be holding my breath.
